test/test_08_minimal_controller_plugin.py

```diff
@@ -0,0 +1,63 @@
+import sys
+import os.path
+from os import environ
+
+_path = os.path.abspath(os.path.join(os.path.abspath(__file__), "../../"))+"/minimal"
+sys.path.append(_path)
+
+from minimal.Simple import self_test, Simple
+from time import sleep
+from ctypes import c_bool
+from multiprocessing import Pool
+from brain.queries import RBO, RBJ
+from brain import connect
+from pytest import fixture
+import docker
+
+
+
+EXT_SIGNAL = c_bool(False)
+CLIENT = docker.from_env()
+
+@fixture(scope='module')
+def rethink():
+    sleep(3) #prior test docker needs to shut down
+    try:
+        tag = environ.get("TRAVIS_BRANCH", "dev").replace("master", "latest")
+    except KeyError:
+        tag = "latest"
+    container_name = "brain_minimal"
+    CLIENT.containers.run(
+        "ramrodpcp/database-brain:{}".format(tag),
+        name=container_name,
+        detach=True,
+        ports={"28015/tcp": 28015},
+        remove=True
+    )
+    yield True
+    # Teardown for module tests
+    containers = CLIENT.containers.list()
```

Comment:
You can just use the previously returned container to stop, i.e. `con.stop(timeout=5)`.